.github/workflows/build-test.yaml

```diff
@@ -20,6 +20,9 @@ jobs:
         shell: bash
 
       - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
```

Comment:
This line should have a comment linking to the document explaining how this can be unsafe